app/controllers/api/v1/offers_controller.rb

```diff
@@ -98,6 +98,15 @@ def complete_review
       render json: @offer, serializer: serializer
     end
 
+    api :PUT, '/v1/offers/1/close_offer', "Mark Offer as closed."
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.